ERCS/erc-7831.md

```diff
@@ -0,0 +1,367 @@
+    "b9a09f33669435E7ef1beAED5aaEB605"
+    "3f3e94c9b9a09F33669435e7ef1beAED"
+)
+```
+
+</details>
+
+For example, these strings are correctly cased:
+
+* `004f67dAbb603AAA58eD52641CCafF09C559704A`
+*  `04f67dAbb603AAA58eD52641CCafF09C559704A`
+*   `4F67dABB603aAa58Ed52641cCAff09C559704A`
+
+[RFC 2119]: https://www.rfc-editor.org/rfc/rfc2119
+[RFC 8174]: https://www.rfc-editor.org/rfc/rfc8174
+[RFC 5234]: https://www.rfc-editor.org/rfc/rfc5234
+[RFC 7405]: https://www.rfc-editor.org/rfc/rfc7405
+[ERC-55]: ./eip-55.md
+[ERC-137]: ./eip-137.md
+[EIP-155]: ./eip-155.md
+[ERC-165]: ./eip-165.md
+[ERC-2304]: ./eip-2304.md
+
+## Rationale
+
+### Component Order
+
+The components are ordered from most specific to most general because... <!-- TODO -->
+
+### Separator Choice
+
+The colon (`:`) is a reasonable choice for separator because it is not an allowed character in ENS names, it is familiar (eg. IPv6), and isn't as overloaded as the `@` symbol.
+
+#### Alternative: `@`
+
+The `@` symbol is a common choice for addresses, and finds use in email and several federated communication protocols. The English reading (foo-**AT**-example-DOT-com) is natural and implies a hierarchy between the left and the right components.
+
+Unfortunately, because the `@` symbol is so widely used, including it in a chain-specific address would make all those protocol identifiers more confusing (or even invalid.) For example, `foo@foo.eth@ethereum` is not a valid email address.
+
+#### Alternative: `/`
+
+<!-- TODO -->
+
+### Target Chain as Subdomain
+
+While it would be technically possible to resolve `chain-part` against a root ENS name (eg. `ethereum.eth` instead of `ethereum.tbd.eth`<!-- TODO -->), using a subdomain allows the pre-registration of well-known chain names for an initial distribution of names before switching to open registration.
+
+Without such a pre-registration, an attacker could register well-known names before the legitimate project.
+
+After the pre-registration period, open registration is acceptable because new chains can register their names before announcing publicly.
+
+## Backwards Compatibility
+
+It is always possible to determine whether a particular string is a chain-specific address, a plain address, or a plain ENS name. Because of this property, there is little opportunity for backwards incompatibility: chain-specific addresses are not valid legacy addresses or ENS names, so tools without support will simply reject them.
+
+## Test Cases
+
+<!--
+  -- TODO: Test Case Ideas
+  --
+  -- * Longer than 20-byte hex local-part
+  -->
+
+### ENS Configuration
+
+#### Mainnet (1)
+
+| Name          | Coin Type       | Record                                       |
+| ------------- | --------------- | -------------------------------------------- |
+| `ethereum.tbd.eth`<!-- TODO --> | - | a bridge contract to mainnet (1)         |
+| `rollup1.tbd.eth`<!-- TODO -->  | - | a bridge contract to rollup1 (1608)      |
```

Comment:
why can't we specify `tbd.eth` as `REGISTRY_SUBDOMAIN` and use later everywhere else

Reply:
We could. Would that be better?

Reply:
yes thats more easy for eyes to identiy ... it took me a while to see tbd.eth was a TBD lol